### ISSUE TITLE
DAC 17 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 04

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#17]
<img width="240" alt="DAC_Colour_Contrast_Non_Text_04" src="https://user-images.githubusercontent.com/75850106/155560497-24fc550d-7b4f-4408-b0b5-12d3b051eb62.png">

### Issue
The map that shows ‘More income-deprived’ areas is highlighted with an orange colour. The orange colour used does not have a minimum contrast of 3:1 against adjacent colours. This means that low vision users may not be able to see the highlighted areas.

<img width="597" alt="DAC 17 Current code ref" src="https://user-images.githubusercontent.com/75850106/155560663-9a0f1701-7e80-49a7-85a5-3c809f964b58.png">

**Low vision comments:**
 “The orange portions of the map fail non-text colour contrast at a ratio of approximately 1.5:1, this should ideally be above 3:1”

### Suggested Solution
Ensure that non-text colours used are meet the minimum 3:1 colour contrast against adjacent colours.
